client/src/main/java/org/opensearch/ml/client/MachineLearningClient.java

```diff
@@ -323,8 +323,21 @@ default ActionFuture<DeleteResponse> deleteConnector(String connectorId) {
         return actionFuture;
     }
 
+    /**
+     * Delete connector for remote model
+     * @param connectorId The id of the connector to delete
+     * @return the result future
+     */
+    default ActionFuture<DeleteResponse> deleteConnector(String connectorId, String tenantId) {
+        PlainActionFuture<DeleteResponse> actionFuture = PlainActionFuture.newFuture();
+        deleteConnector(connectorId, tenantId, actionFuture);
+        return actionFuture;
+    }
+
     void deleteConnector(String connectorId, ActionListener<DeleteResponse> listener);
```

Comment:
Do we want this one to delegate to the other one with a default (null) tenant ID?